cypress/support/ui_commands.js

```diff
@@ -5,6 +5,19 @@
 
 import * as TIMEOUTS from '../fixtures/timeouts';
 
+Cypress.Commands.overwrite('get', (originalFn, selector, options) => {
+    function isCssSelector(string) {
+        const regex = /[a-z]*?(@|#|\.).+/;
```

Comment:
This is way not enough to parse CSS selectors. nestings(`.parent > .child`), combinations(`[attr=val]`), etc... Maybe we need to use 3rd party css parser library.

I suggest to define new command `getByTestId()` just like react-testing-library does.

Reply:
I'll make it an alternate command for now. We may want to look at just using cypress-testing-library to get `getByTestId()` for free. I'm thinking I should just update that `data-` attribute to just be `data-testid` so we are somewhat future proofed then.

Another approach I was thinking, and I'd love your input, was to preface our testId selectors with our own symbol, like `cy.get('@my-test-id')`. On one side of the coin, I like the simplicity of having just on command ,`get`, to handle getting elements. But on the other we may lose clarity and add a certain level of "magic".

Reply:
I think conventions that are not properly guided are literally "magic". In that sense, I usually (include this case) prefer the API approach.

Reply:
And also using `.get('@...')` would be too easy to conflict with regular aliases.

Reply:
`@` was a bad example then :)

Separate command it is!